wgsl/index.bs

```diff
@@ -1971,8 +1971,8 @@ then:
 
 When a value of structure type |S| is placed at byte offset |k| of a host-shared memory buffer,
 then:
-   * The |i|'<sup>th</sup> member of the structure value is placed at byte offset |k| + [=OffsetOfMember=](|S|,|i|)
-
+   * The |i|'<sup>th</sup> member of the structure value is placed at byte offset |k| + [=OffsetOfMember=](|S|,|i|).
```

Comment:
Is it worth mentioning only `k`s that do not violate the alignment of the structure are valid?

Reply:
This is true for all k's for all the types.  I'll add the general statement.